spring-cloud-generator/src/test/java/com/google/cloud/generator/spring/utils/RestTestProtoLoader.java

```diff
@@ -0,0 +1,124 @@
+// Copyright 2021 Google LLC
```

Comment:
```suggestion
// Copyright 2023 Google LLC
```

Reply:
This helper was also copied (source: [RestTestProtoLoader](https://github.com/googleapis/sdk-platform-java/blob/main/gapic-generator-java/src/test/java/com/google/api/generator/test/protoloader/RestTestProtoLoader.java)), and soon to be removed when the next gapic-generator-java release publishes the testlib. Would this warrant a header year update, or should the same copyright year be kept as source?

Reply:
I see, I think it makes sense to keep it as it is then.